composer.json

```diff
@@ -10,20 +10,23 @@
         }
     ],
     "require": {
+        "ezsystems/ezpublish-kernel": "~5.1 || ^6.7",
         "guzzlehttp/guzzle": "~5.0|~6.0",
         "components/handlebars.js": "~3.0.0",
-        "symfony/symfony": "^2.7.7"
+        "symfony/symfony": "^2.7.7 || ^3.3"
     },
     "require-dev": {
-        "ezsystems/ezpublish-kernel": "~6.0.0 | ~5.1",
```

Comment:
Moved this up to `require`. It makes no sense here to have it in `-dev` as this is not test code.

@damianz5 @SylvainGuittard was there any reason to support kernel `>=6.0.0 <6.1.0` only? Above, I bumped constraint to LTS: `^6.7`.

Reply:
Ok, I think I know the reason to use very early version of Kernel. Fixed it af6d17d
Note that `EzSystems\RecommendationBundle\Client\YooChooseNotifier` depends on `eZ\Publish\Core\SignalSlot\Repository` which is terribly wrong, but a little bit out of scope here :)